lib/models.js

```diff
@@ -1242,9 +1500,19 @@ class TBD extends Value{
   toString() {
     return this._text ? `TBD(${this._text})` : 'TBD';
   }
+ 
+  toJSON() {
+    return Object.assign(
+      {
+        "fqn" : this.text
```

Comment:
oooh...  do you think it's right to assign the TBD _text_ to `fqn`?  It could be anything (and even include spaces I think).

Reply:
To be honest, I didn't know how to properly handle it. It's not a proper `fqn`, but it's the closest thing to it for a TBD value.

I've changed it to `"fqn" : this.toString()`, which comes from
```
  toString() {
    return this._text ? `TBD(${this._text})` : 'TBD';
  }
```

It's not much better, so if you have a better idea, I can update it again.

Reply:
(Needs review)

Reply:
I don't have a better solution.  Let's just roll with this for now.